src/stage0.txt

```diff
@@ -25,7 +25,7 @@ cargo: 0.38.0
 #
 # This means that there's a small window of time (a few days) where artifacts
 # are downloaded from dev-static.rust-lang.org instead of static.rust-lang.org.
-# In order to ease this transition we have an extra key which is in the 
+# In order to ease this transition we have an extra key which is in the
```

Comment:
I feel like this happens every time I edit this file... probably because we keep re-getting this from master or something.

Reply:
Maybe send a PR to master with that change?